Documentation/helm-operator.md

```diff
@@ -46,47 +39,21 @@ The release channel is the most recent release of Rook that is considered stable
 
 ```console
 helm repo add rook-release https://charts.rook.io/release
-```
-
-For Helm `v3.x`:
-
-```console
 helm install --namespace rook-ceph rook-ceph rook-release/rook-ceph
 ```
 
-For Helm `v2.x` the `--name` flag needs to be specified:
-
-```console
-helm install --namespace rook-ceph --name rook-ceph rook-release/rook-ceph
-```
-
-### Master
+### Updating Custom Resource Definitions (CRDs)
 
-The master channel includes the latest commits, with all automated tests green. Historically it has been very stable, though it is only recommended for testing.
-The critical point to consider is that upgrades are not supported to or from master builds.
-
-To install the helm chart from master, you will need to pass the specific version returned by the `search` command.
+Rook's CRDs must be updated separately from Rook's helm chart.
+The initial install will create the CRDs, but Helm does not support updating CRDs.
+When upgrading the Rook Helm chart, you will also need to apply the latest CRDs.
 
 ```console
-helm repo add rook-master https://charts.rook.io/master
-
-# For Helm v3.x
-helm search repo rook-ceph --versions
-helm install --namespace rook-ceph rook-ceph rook-master/rook-ceph --version <version>
-
-# For Helm v2.x
-helm search rook-ceph
-helm install --namespace rook-ceph --name rook-ceph rook-master/rook-ceph --version <version>
-```
-
-For example to install version `v1.3.0.860.g80ff2bb`:
-
-```console
-helm install --namespace rook-ceph rook-ceph rook-master/rook-ceph --version v1.3.0.860.g80ff2bb
+git clone --single-branch --branch {{ branchName }} https://github.com/rook/rook.git
```

Comment:
Requiring the user to clone the git repo and also do work to ensure they are using the right branch makes it quite a bit more difficult to manage Rook using Helm. Is it possible to instead run `kubectl apply -f crd.yaml` from the helm chart they intend to use?

The [Helm Pull docs](https://helm.sh/docs/helm/helm_pull/) say that it's possible to pull a chart and extract it locally. I think maybe this workflow, while possibly strange, at least makes it easier for the user to be sure they are matching the CRDs to the chart version more easily.

Reply:
Yes, per discussion on #4037 I'm also looking into that possibility. It will work much smoother if we can just extract it from the chart.